src/Type/JustNullableTypeTrait.php

```diff
@@ -117,4 +117,9 @@ public function isVoid(): TrinaryLogic
 		return TrinaryLogic::createNo();
 	}
 
+	public function isScalar(): TrinaryLogic
+	{
+		return TrinaryLogic::createNo();
```

Comment:
I'd remove this from the trait. Where it's going to be missing, I'd just add it there in a specific class.

Reply:
adapted, makes sense. that trait is a bit weird actually, yeah. thought the same when implementing it, but went with "consistency" with other is* methods instead. reminds me also that the traits still need cleaning-up and a rule that keep them clean :/